README.md

```diff
@@ -102,15 +102,14 @@ Available targets:
 
 | Name | Version |
 |------|---------|
-| terraform | >= 0.12.0, < 0.14.0 |
```

Comment:
we need to add `README.yaml`.
we generate `README.md` from it.
See https://github.com/cloudposse/terraform-aws-datadog-integration/blob/master/README.yaml